recipe/bld.bat

```diff
@@ -0,0 +1,22 @@
+if "%ARCH%"=="32" (
+   set MACHINE="IX86"
+   :: A different SDK is needed when build with VS 2017 and 2015
+   :: http://wiki.tcl.tk/54819
+   if "%VS_MAJOR%"=="14" (
+    echo "Switching SDK versions"
+    call "%VS140COMNTOOLS%..\..\VC\vcvarsall.bat" x86 10.0.15063.0
+   )
+) else (
+  set MACHINE="AMD64"
+  :: A different SDK is needed when build with VS 2017 and 2015
+  :: http://wiki.tcl.tk/54819
+  if "%VS_MAJOR%"=="14" (
+    echo "Switching SDK versions"
+    call "%VS140COMNTOOLS%..\..\VC\vcvarsall.bat" x64 10.0.15063.0
```

Comment:
this could potentially render the binary unusable in our envs, since conda-forge builds differently from us. it would be good to do some manual testing to verify it works with other packages.

Reply:
I removed this code. It looks like it was fixed by the upstream https://core.tcl-lang.org/tk/info/45c1899a21ea0fe2